### PR TITLE
feat(cli): intro warning (28) support for inherited tables

### LINKED ISSUE #17783: Handle new introspection warnings for partitioned tables in CLI
Currently:
```
PS C:\Users\Jan\Documents\throwaway\partitionedTables> npx prisma@4.10.0-integration-engines-4-10-0-83-integration-evie-partition-tables-fix-0c08c4bf9d6ec24df1142c87016ffac6ca85b118.1 db pull
Need to install the following packages:
  prisma@4.10.0-integration-engines-4-10-0-83-integration-evie-partition-tables-fix-0c08c4bf9d6ec24df1142c87016ffac6ca85b118.1
Ok to proceed? (y) y
Prisma schema loaded from prisma\schema.prisma
Environment variables loaded from .env
Datasource "db": PostgreSQL database "salmon_marmot", schema "public" at "db-provision-postgres23452b4.c8yxynpcltwd.us-east-1.rds.amazonaws.com:5432"

✔ Introspected 2 models and wrote them into prisma\schema.prisma in 3.48s

*** WARNING ***

The following models were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client.
- "measurement"

These tables are partition tables, which are not yet fully supported.
Code 27
[
  {
    "model": "blocks"
  },
  {
    "model": "measurement"
  }
]

Run npx prisma generate to generate Prisma Client.
```

This CLI output should be formatted properly.

### LINKED ISSUE #18704: Render warning code 28 in cli (PG Inherited Tables)
https://github.com/prisma/schema-team/issues/396

### PR DESCRIPTION
**Pull request by Druue**
closes #18704
ON HOLD per [slack](https://prisma-company.slack.com/archives/C04K84V31T4/p1681221108593119?thread_ts=1679339266.495589&cid=C04K84V31T4)

### CODE REVIEW

File: packages/migrate/src/commands/DbPull.ts [hunk 1]
```diff
@@ -480,6 +480,8 @@ ${`Run ${chalk.green(getCommandWithExecutor('prisma generate'))} to generate Pri
           message += warning.affected.map((it) => `- View "${it.view}", Field: "${it.field}"`).join('\n')
         } else if ([23, 24, 25].includes(warning.code)) {
           message += warning.affected.map((it) => `- View "${it.view}"`).join('\n')
+        } else if (warning.code === 28) {
+          message += warning.affected.map((it) => `- Model "${it.model}"`).join('\n')
```
Comment (janpio) [inline packages/migrate/src/commands/DbPull.ts:484]: Couldn't this be another case for line 461?

File: packages/migrate/src/__tests__/DbPull.test.ts [hunk 2]
```diff
@@ -1170,15 +1170,191 @@ describe('postgresql views introspection warnings', () => {
...
+        // These tables are inherited tables, which are not yet fully supported.
+        // - Model "cities"
+        // 
+      `)
+      expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+  })
+
+  describe('postgresql inhereited 28/2 - inherited tables found', () => {
```
Comment (janpio) [inline packages/migrate/src/__tests__/DbPull.test.ts:1287]: What is this really testing? From a quick look, it seems more like the same but with the tables duplicated?

File: packages/migrate/src/__tests__/DbPull.test.ts [hunk 3]
```diff
@@ -1170,15 +1170,191 @@ describe('postgresql views introspection warnings', () => {
...
+        }
+
+        model definitely_not_capitals {
+          name       String?
+          population Float?  @db.Real
+          elevation  Int?
+          state      String  @unique @db.Char(2)
+        }
+
+        /// This table has subclasses and requires additional setup for migrations. Visit https://pris.ly/d/table-inheritance for more info.
```
Comment (janpio) [inline packages/migrate/src/__tests__/DbPull.test.ts:1331]: This link does not work yet @eviefp

File: packages/migrate/src/__tests__/DbPull.test.ts (hunk 2 above)
Comment (Druue) [reply packages/migrate/src/__tests__/DbPull.test.ts:1287]: It's just effectively making sure that stuff is being formatted correctly for when we detect multiple -- off the back of what happened previously 😅 but it works so am also happy to just not leave this in

File: packages/migrate/src/commands/DbPull.ts (hunk 1 above)
Comment (Druue) [reply packages/migrate/src/commands/DbPull.ts:484]: Ope, yeah, it seems to be :) I'll update that

File: packages/migrate/src/__tests__/DbPull.test.ts (hunk 2 above)
Comment (janpio) [reply packages/migrate/src/__tests__/DbPull.test.ts:1287]: No problem - just the name might explain it a bit better.

**Comment by Jolg42** [comment]
Not needed since https://github.com/prisma/prisma/pull/18869

**Pull request closed without merging**